digdag-ui/model.js

```diff
@@ -297,6 +297,15 @@ export class Model {
     })
   }
 
+  retrySession (session: Session) {
+    const { lastAttempt } = session
+    return this.put('attempts', {
```

Comment:
@frsyuki upon submission (and api response) the page redirects to `/session` so double click should not be a possible case